Cargo.lock

```diff
@@ -767,7 +781,7 @@ checksum = "4569e456d394deccd22ce1c1913e6ea0e54519f577285001215d33557431afe4"
 dependencies = [
  "hermit-abi",
  "libc",
- "wasi",
+ "wasi 0.11.0+wasi-snapshot-preview1",
```

Comment:
Same minor version as Firefox:

https://github.com/mozilla-firefox/firefox/blob/727f98387cec6b42f169f567748c167202b3dc03/Cargo.lock#L7449-L7454

(Test code only for us.)